app/views/layouts/_support_links.html.erb

```diff
@@ -0,0 +1,5 @@
+<h2 class="govuk-heading-m">Support and guidance</h2>
+<p class="govuk-body-s">
+  If you have a question, or you've had a problem using this service, please contact us at
```

Comment:
```suggestion
  If you have a question, or you‘ve had a problem using this service, please contact us at
```

Reply:
Sorry my eyesight! - wrong smart quote

Reply:
for what it's worth, I copied this from the other repo!